data-jdbc/src/test/groovy/io/micronaut/data/jdbc/h2/one2one/OneToOneSpec.groovy

```diff
@@ -55,17 +55,17 @@ DROP TABLE IF EXISTS `TestXyzCustomerDetails`;
 
 CREATE OR REPLACE TABLE `TestXyzCategory` (
   `id` BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
-  `active` bit(1) DEFAULT NULL,
+  `active` boolean DEFAULT NULL,
```

Comment:
These are also changes needed since h2 version used in prepare-for-mn4 does not support previous column definitions. Similar was needed in table creation for array fields.